tests/periph_pm/Makefile

```diff
@@ -0,0 +1,10 @@
+BOARD ?= slstk3401a
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED += periph_pm
```

Comment:
I don't think this is needed, this feature is made optional (see Makefile.dep). And that's probably why the CI is failing.

Reply:
Fixed.